{{cookiecutter.project_slug}}/merge_production_dotenvs_in_dotenv.py

```diff
@@ -4,13 +4,13 @@
 
 import pytest
 
-ROOT_DIR_PATH = Path(__file__).parent.resolve()
-PRODUCTION_DOTENVS_DIR_PATH = ROOT_DIR_PATH / ".envs" / ".production"
+BASE_DIR_PATH = Path(__file__).parent.resolve()
```

Comment:
Hum... I know it was a problem before, but I just noticed that this variable is inconsistent with the others. I think we could rename it.

```suggestion
BASE_DIR = Path(__file__).parent.resolve()
```

Reply:
Actually, there are a few other things we can improve, let's do it as a follow up.